pkg/server/ca/rotator/rotator_test.go

```diff
@@ -588,3 +617,11 @@ func createSlot(id string, now time.Time, hasValue bool) *fakeSlot {
 		isActive:        hasValue,
 	}
 }
+
+func init() {
```

Comment:
can you add comment about this is going to be removed from 1.9?

Reply:
This PR has been moved to 1.9, so I'm removing the feature flag.